index.html

```diff
@@ -0,0 +1,154 @@
+         
+          <div class="humgurger">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+          </div>
+        </nav>
+      </div>
+      <div class="row introduction">
+        
+        <p>"Welcome to Healthline Nutrition!"</p>
+        <h1 class="secondHeader">
+          The Definitive Guide<br />
+          to Healthy Eating in Real Life
+        </h1>
+
+        <p id="second-p">
+          healthy eating doesn’t have to be complicated. It’s entirely possible
+          to nourish your body while enjoying the foods you love.
+        </p>
+        <img src="assets/images/Health-Nutrition.jpg" alt="Nutrition" />
+        
+      </div>
+      <div class="row mainProgram" id="program">
+        <h3>Main Program</h3>
+        <hr />
+        <div class="col-sm card">
+          <img src="assets/images/vegtables.jpg" alt="Vegtables" />
+          <h4 class="card-title">Vegetables</h4>
+          <p class="card-text">
+            Add variety to grilled or steamed vegetables with an herb such as
+            rosemary..
+          </p>
+        </div>
+        <div class="col-sm card">
+          <img src="assets/images/fruits.jpg" alt=" fruits" />
+          <h4 class="card-title">Fruits</h4>
+          <p class="card-text">
+            Fresh, frozen, or canned fruits are great choices. Try fruits beyond apples and bananas such as mango, pineapple or kiwi fruit
+          </p>
+        </div>
+        <div class="col-sm card">
+          <img
+            src="assets/images/calcium.jpg"
+            alt=" calcuim-rich food image"
+          />
+          <h4 class="card-title">calcium-Rich food</h4>
+          <p class="card-text">
+            In addition to fat-free and low-fat milk, consider low-fat and fat-free yogurts without added sugars.
+          </p>
+        </div>
+        <div class="col-sm card">
+          <img src="assets/images/meat.jpg" alt=" meat picture" />
+          <h4 class="card-title">Meat</h4>
+          <p class="card-text">
+            If your favorite recipe calls for frying fish or breaded chicken, try healthier variations by baking or grilling. 
+          </p>
+        </div>
+
+        <p class="text-white" id="wholeProgram">SEE THE WHOLE PROGRAM</p>
+      </div>
+      <div class="row">
+      <section class="sports">
+        <h2>Sports</h2>
+        <hr>
+      </section>
+      
+      </div>
+      <div class="row">
+        <section class="partner">
+            <h3>Parnter</h3>
+            <hr>
+            <span class="mozilla">
+             mozilla
+            </span>
+            <span class=" google">
+              Google
+            </span>
+            <span class="naver">
+              Naver
+            </span>
+            <span  class="daumkakao">
+              daumkakao
+            </span>
+            <span  class="airbnb">
+              airbnb
+            </span>
+        </section>
+      </div>
```

Comment:
- [ ] Please move this section after the 'featured speakers' section and kindly use the colors suggested, as shown in the template. 

<img width="1361" alt="Screen Shot 2022-09-12 at 11 06 28 AM" src="https://user-images.githubusercontent.com/83717285/189702582-f8e9d058-8e64-4485-bff9-0e25575f996b.png">

Reply:
@smunozmo can you please give me a hint how to fix this issue ?

Reply:
Move this section after Sports 'Dynamic page'

Reply:
Hi @ManarOmari, sure:
What I mean is that the 'partners' section should be before the footer and after your 'sports' section as in the template:

<img width="1045" alt="Screen Shot 2022-09-13 at 8 42 10 AM" src="https://user-images.githubusercontent.com/83717285/189917775-c763517d-35b0-4911-a43b-f5979bc85a66.png">

Right now your 'partners' section is displayed before your 'sports' section:

<img width="1352" alt="Screen Shot 2022-09-13 at 8 44 10 AM" src="https://user-images.githubusercontent.com/83717285/189918273-a2678894-dccb-41f7-b699-556dc1dca214.png">

Just keep the sections in the order suggested by the template and you will be fine 😎

Reply:
@smunozmo  Thank you . But I mean the page always appears on the last not in section Sports . Can you give me hint how it will appear in correct position ?

Reply:
@ManarOmari no problem, I would recommend checking what is this line doing:

https://github.com/ManarOmari/CapstoneProject1/blob/671f0365fb7e07c60f2ecca1be44c36c96992f53/sport.js#L68

Reply:
I solved it and now it appears on correct order :D